Baekjoon/5585.py

```diff
@@ -0,0 +1,15 @@
+'''
+문제 링크: https://www.acmicpc.net/problem/5585
+카테고리: 그리디 알고리즘
+문제 해설: 기본 거스름돈 구하기
+'''
+def main(a):
+    a=1000-a
+    coin=[500,100,50,10,5,1]
+    count=0
+    for i in coin:
+        count+=a//i
+        a%=i
+    return count
```

Comment:
```suggestion
def main(a):
    b=1000-a
    coin=[500,100,50,10,5,1]
    count=0
    for i in coin:
        count+=b//i
        b%=i
    return count
```

입력으로 들어온 a를 변형 안시키고, b라는 변수에 새로 할당하여 b를 조작하는 코드도 좋을 것 같습니다

Reply:
알겠습니당